### PR TITLE
add cropit@0.5.1 w/ git auto-update

### LINKED ISSUE #9647: [Request] Add cropit
**Library name:** cropit
**Git repository url:** https://github.com/scottcheng/cropit
**npm package url(optional):** https://www.npmjs.com/package/cropit
**License(s):** https://github.com/scottcheng/cropit/blob/master/LICENSE
**Official homepage:** http://scottcheng.github.io/cropit/
**Wanna say something? Leave message here:** 

**What it is**

- A jQuery plugin for image cropping and zooming.
- Loads images locally via FileReader, and crops them using canvas.
- Designed to be extremely customizable via CSS.
- Best for the cases where you want users to upload images of a specific size and aspect ratio.

**Features**

- Works on desktop and mobile with touch support (except IE 9 and below).
- Drag'n'drop image into cropper.
- Extremely performant even with huge images.
- Extensive options and APIs to give you full control.

=====================
Notes from cdnjs maintainer:
You are welcome to add a library via sending pull request,
it'll be faster then just opening a request issue,
and please don't forget to read the guidelines for contributing, thanks!!

### PR DESCRIPTION
**Pull request by clairetsai818**
Pull request for issue: #9647  
Related issue(s): #9414 

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required): https://github.com/scottcheng/cropit
 * Official website (optional, not the repository): http://scottcheng.github.io/cropit
 * NPM package url (optional): https://www.npmjs.com/package/cropit
 * GitHub / Bitbucket popularity (required):
   - Count of stars: 630
   - Count of watchers: 40
   - Count of forks: 232
 * NPM download stats (optional):
   - Downloads in the last day: 123
   - Downloads in the last week: 947
   - Downloads in the last month: 2530

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [x] The lib has notable popularity
   * [x] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [x] More than 500 downloads stats per month on npm registry
 * [x] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [x] Has valid tags for each versions (for git auto-update)
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

close #9647, cc @scottcheng

### CODE REVIEW

**Comment by clairetsai818** [comment]
@PeterDaveHello Please help me review it, thanks.

File: ajax/libs/cropit/package.json [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+        "files": [
+          "**/*"
+        ]
+      }
+    ]
+  },
+  "dependencies": {
+    "jquery": ">=1.9"
+  },
+  "readmeFilename": "README.md"
```
Comment (PeterDaveHello) [inline ajax/libs/cropit/package.json]: Please remove this part (readmeFilename)

File: ajax/libs/cropit/package.json [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+    "name": "Scott Cheng",
+    "email": "me@scottcheng.com",
+    "url": "http://scottcheng.com"
+  },
+  "keywords": [
+    "crop",
+    "zoom",
+    "image editor"
+  ],
+  "homepage": "http://scottcheng.github.io/cropit",
```
Comment (PeterDaveHello) [inline ajax/libs/cropit/package.json]: Let's use `https` instead of `http` here.

**Comment by PeterDaveHello** [comment]
@clairetsai818 once you fixed it, please also update your local master branch, and do this steps to update your PR branch:`git rebase master cropit`,then do a force push to update the PR:

**Comment by clairetsai818** [comment]
@PeterDaveHello I've revised the file. Please help me review it, thanks.

**Comment by PeterDaveHello** [comment]
@clairetsai818 Please rebase your branch with our latest master branch, thanks.

**Comment by clairetsai818** [comment]
@PeterDaveHello I've rebased my branch with the latest master branch. Please help me review it, thanks.

**Comment by PeterDaveHello** [comment]
@clairetsai818 Looks like you didn't, are you sure about that you updated your local master branch?

**Comment by clairetsai818** [comment]
@PeterDaveHello I've rebased my branch with the latest master branch. Please help me review it, thanks.

**Comment by PeterDaveHello** [comment]
Thanks for your contribution @clairetsai818

**Pull request merged**